src/utilities/helperFunctions.js

```diff
@@ -62,6 +62,78 @@ export const getIconSrc = function getWeatherIconSrcURL(iconId) {
   return `https://openweathermap.org/img/wn/${iconId}@2x.png`;
 };
 
+export const getIconName = function matchIconNameWithID(
+  { id, icon },
+  animated = false
+) {
+  const iconMatches = [];
```

Comment:
Does iconMatches need to be an array? Isn't iconMatches always gonna be either empty or have a single value (ThunderstomRain, Drizzle, Sleet, etc)?

Reply:
Some icons are paired with two names since there is a day/night alternative.

Reply:
Oh okay I see, that strange that the API would return two names for day/night alternatives. I would assume it would just display a single icon to represent the current weather.

Reply:
I think it must be time-based, because yeah it is strange.

Reply:
"weather": [
        {
          "id": 500,
          "main": "Rain",
          "description": "light rain",
          "icon": "10d"
        }
]

For example, this is from the daily weather api call. This array could have two icons in it? So another element would be present?

Reply:
I don't think the call would ever have two icons, I'm thinking it passes a different icon code based on the timestamp.

<img width="690" alt="image" src="https://github.com/awong-pdx/frontend-weather-dashboard/assets/74124296/b3620185-915c-47bb-b3fb-ac9824e6ddc0">

Reply:
The icons are group by number so 5xx is rain, 3xx is drizzle, etc. But it seems like group 800 and group 80x are the only ones that have two possible icon to represent day and night. In that case, I believe it only returns one or the other. If the timestamp represent the morning, the icon would be 01d, but at night, it would be 01n. Is that correct?

Reply:
That sounds right.

Reply:
If that's the case, then would the iconMathces array be needed?

Reply:
Yes, because, for example, `id`=800 matches with both `ClearDay` and `ClearNight`
 
<img width="247" alt="image" src="https://github.com/awong-pdx/frontend-weather-dashboard/assets/74124296/327b5c12-265b-41a1-bbad-f838810b81e1">

Reply:
Ahh okay I see, that clarifies it. It's really only for a couple of icons where there are multiple matches!

Reply:
That makes sense, thanks for clarifying! It seems this is only the case for all the 800 codes.